css/style.css

```diff
@@ -0,0 +1,200 @@
+:root{
+    --color-nav: #be3144;
+    --color-project:#45567d;
+    --color-white: #f0f0f0;
+    --color-grey: #303841;
+
+    /* shadows */
+    --shadow-nav: 0 2px 1 rgba(0, 0, 0, .4);
+    --shadow-project: 1px 1px 2px rgba(0, 0, 0, .5);
+    --shadow-text: 2px 2px 1px #1f1f1f;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+*,
+*::before,*::after {
+    box-sizing: inherit;
+}
+
+html {
+    font-size: 62.5%; 
+}
+
+body {
+    font-family: sans-serif;
+    font-size: 1.8rem;
+    font-weight: 400;
+    line-height: 1.4;
+    color: var(--color-white);
+}
+
+.nav {
+    display: flex;
+    justify-content: flex-end;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    background-color: var(--color-nav);
+    z-index: 100;
+    box-shadow: var(--shadow-nav);
+}
+
+.nav-list {
+    display: flex;
+    margin-right: 2rem;
+}
+
+.nav-list a {
+    display: block;
+    padding: 2rem;
+    font-size: 2.2rem;
+}
+
+a {
+    text-decoration: none;
+    color: var(--color-white);
+}
+
+a:link {
+    cursor: pointer;
+}
+
+ul {
+    list-style: none;
+}
+
+li {
+    display: list-item;
+    text-align: center;    
+}
+
+.welcome-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100vh;
+    background-color: #000;
+    background-image: linear-gradient(62deg, #3a3d40 0%, #181719 100%);
+}
+
+h1 {
+    font-size: 6rem;
+    display: block;
+}
+
+.welcome-section >p {
+    font-size: 3rem;
+    font-weight: 200;
+    font-style: italic;
+    color: var(--color-nav);
+}
+
+.projects-section{
+    text-align: center;
+    padding: 10rem 2rem;
+    background-color: var(--color-project);
+}
+
+.projects-grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+    grid-gap: 4rem;
+    width: 100%;
+    max-width: 1280px;
+    margin: 0 auto;
+    margin-bottom: 6rem;
+}
+
+.project-title {
+    font-size: 2rem;
+    padding: 2rem .5rem;
+}
+
+.project {
+    background-color: var(--color-grey);
+    box-shadow: var(--shadow-project);
+}
+
+h2 {
+    font-size: 4.2rem;
+    display: block;
+}
+
+h1, h2 {
+    font-family: 'Raleway', sans-serif;
+    text-align: center;
+    font-weight: 700;
+}
+
+section {
+    display: block;
+}
```

Comment:
Please use class attributes like `.title` rather than using tag names like `h1`, using an appropriate class name in CSS will make your code easier to read and maintain.